### PR TITLE
OWASP check profile + fixed CVEs in some of the connectors

### PR DESCRIPTION
**Pull request by dlg99**
Added OWASP check profile.

Fixed CVEs at the connectors for: 
* Big Query
* Azure Kusto
* JMS

### CODE REVIEW

**Review by eolivelli — APPROVED** [review]
Lgtm 

We are updating protobuf and jetty, there is a risk that something breaks.
I wonder how we could setup automated testing for this repo (the main problem is that it needs accounts for services..)

File: pom.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
     <puppycrawl.checkstyle.version>8.37</puppycrawl.checkstyle.version>
     <dockerfile-maven.version>1.4.13</dockerfile-maven.version>
     <typetools.version>0.5.0</typetools.version>
-    <protobuf3.version>3.11.4</protobuf3.version>
+    <protobuf3.version>3.16.1</protobuf3.version>
     <protoc3.version>${protobuf3.version}</protoc3.version>
     <grpc.version>1.33.0</grpc.version>
```
Comment (nicoloboschi) [inline pom.xml:89]: I suggest to upgrade grpc to 1.42.1 as LS280

**Review by nicoloboschi — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (dlg99) [reply pom.xml]: Upgraded + everything else to match LS 2.8

**Comment by dlg99** [comment]
@eolivelli most of the dependencies affect the KCA/pulsar parts.
There is a risk for the jms connector but it is not tested yet anyway. 
t already was in a weird state with some of the jetty-related deps coming from an older version, I forced the, to be the same.
In case something does not work I'll have to deal with it anyway, potentially by shading it.

The tests are hard to add and readme for the project calls it out. 
It's possible that I'll invest some time into tests for some connectors, but that won't happen immediately, will require mocks and non-trivial amount of work (see snowflake connector for example)

**Pull request merged**